single_line_class.rb

```diff
@@ -0,0 +1,27 @@
+#Prefer a single-line format for class definitions with no body
+class FooError < StandardError
+end
+
+class Test
+	# Avoid single-line methods.
+    def too_much; puts 'pravin'; puts 'mishra' end 
+    
+    # bad
+    e = M * c ** 2
+    [ 1, 2, 3 ].size
+
+    # bad
+	case
+  		when song.name == 'Misty'
+    		puts 'Not again!'
+  		when song.duration > 120
```

Comment:
__[rubocop]__
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Indent `when` as deep as `case`. [:ref](https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)